mobile/src/main/java/org/hawkular/client/android/activity/DrawerActivity.java

```diff
@@ -16,9 +16,6 @@
  */
 package org.hawkular.client.android.activity;
 
-import java.util.ArrayList;
```

Comment:
@garg-anuj 
I have import settings.jar to the studio, but it is not working. For now, I have to remove the issue manually.